scripts/restart-wrapper.sh

```diff
@@ -25,4 +25,7 @@ do
     child=$!
     sleep 5
     echo "killing pid $child..." && sleep 5 && kill -9 "$child"
+    while kill -0 "$child" 2> /dev/null; do # wait for child to exit (kill -0 is falsy if pid is gone)
```

Comment:
Sometimes it would take a few ms for the process to exit, and we'd try to start a new one before it was dead, which created a port clash. This makes sure we wait for it to exit.